AssignmentJenkins/Pipeline-DSL-Script/Pipeline_dsl.groovy

```diff
@@ -0,0 +1,7 @@
+pipelineJob('Pipeline_DSL') {
+    definition {
+        cps {
+            script(readFileFromWorkspace('File_Read.groovy'))
```

Comment:
what is this cps is doing?

Reply:
CPS stands for continuous passing style. It allows to write script and it allows to pass the file.